lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -63,6 +63,18 @@
 '''
 
 EXAMPLES = '''
+
+# Gather facts
```

Comment:
Can you re-phrase this as 

Gather facts from standalone ESXi server having datacenter as 'ha-datacenter'